### DISCUSSION TITLE
Styles are not minified in production

### DISCUSSION THREAD

**Discussion by matiastucci** [vercel/next.js#17206] [Help]
This is my `next.config.js`

```js
const withCSS = require('@zeit/next-css')
const withLess = require('@zeit/next-less')
const withSass = require('@zeit/next-sass')

if (typeof require !== 'undefined') {
  require.extensions['.less'] = () => {}
}

module.exports = withCSS(
  withLess(
    withSass({
      lessLoaderOptions: {
        javascriptEnabled: true,
      },
    })
  )
)
```

any ideas? 🤔 

Thanks!

**Comment by jamesmosier** [comment]
Have you tried Next's built in CSS support? https://nextjs.org/docs/basic-features/built-in-css-support

**Reply by jamesmosier** [reply]
`const withCSS = require('@zeit/next-css')` and `const withSass = require('@zeit/next-sass')` is not Next's built in support for CSS. This is an older plugin that is not recommended anymore. Next supports both CSS and SASS without any additional plugins. If you want to use LESS you will need the plugin like you have.

Take a look at https://nextjs.org/docs/basic-features/built-in-css-support and it'll give some more information! but basically you dont need to do anything, just add you CSS 😄

**Reply by matiastucci** [reply]
The problem is that I'm using ant.design and in order to override their variables I need to support LESS 😢 and this is the way I found to do that. If I can't stop using LESS what can I do with my current config? Thanks! 🙏

**Comment by matiastucci** [comment]
I think I'm already using that 🤔 or what do you mean?

**Comment by shubh-mehrotra** [comment]
any update on this?
